src/DelegateInterface.php

```diff
@@ -2,17 +2,17 @@
 
 namespace Interop\Http\Middleware;
 
-use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
 
 interface DelegateInterface
 {
     /**
-     * Dispatch the next available middleware and return the response.
+     * Process an incoming server request and return a response.
```

Comment:
Some little remarks:
* The wording was borrowed from `ServerMiddlewareInterface::process`
* About *Dispatch*: This sounds asynchronous to me, but maybe I'm wrong
* About *middleware*: see [above](https://github.com/http-interop/http-middleware/pull/29#issuecomment-260214510)

However, the wording is currently not such important to me – if it speeds up some things.